.github/workflows/dev_tag_retrieval_pr.yaml

```diff
@@ -0,0 +1,88 @@
+name: Dev Tag Retreival PR Generator
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "!*-*"
+jobs:
+  gen_dev_tag_retrieval_pr:
+    name: Generate Dev Tag Retrieval PR
+    # Only run this job if we're in the main repo, not a fork.
+    if: github.repository == 'vmware-tanzu/tanzu-framework'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+
+      - name: Fetch current version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Sanitize to remove 'v' prefix
+        id: get_sanitized_version
+        run: echo '##[set-output name=SANITIZED_VERSION;]$(echo ${{ steps.get_version.outputs.VERSION }} | sed 's/v//')'
+
+      - name: Increment Version to dev
+        id: next_version
+        run: |
+          IFS='.' read -ra ver <<< "${{ steps.get_sanitized_version.outputs.SANITIZED_VERSION }}"
```

Comment:
Given that `major` release bump is not deterministic (would be done manually), we don't need to remove `v` since we'll alway be bumping only `minor` verion.

Reply:
Fixed with commit  fbc99d43bbf97a1dcd9904e62e2f28e338d3735f (https://github.com/vmware-tanzu/tanzu-framework/pull/1190/commits/fbc99d43bbf97a1dcd9904e62e2f28e338d3735f)

Reply:
Also updated using 1be88ae08c815edb06ae8b464d4294b4f0efe00c (https://github.com/vmware-tanzu/tanzu-framework/pull/1190/commits/1be88ae08c815edb06ae8b464d4294b4f0efe00c)